package.json

```diff
@@ -252,6 +252,6 @@
     "uswds": "1.4.2",
     "vanilla-lazyload": "^8.17.0",
     "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#d8bac128afcb5553667d3d8237a71344aa7d2ade",
-    "webpack-bundle-analyzer": "^2.11.1"
+    "whatwg-fetch": "^2.0.3"
```

Comment:
I may be seeing this prematurely, but was this intentional?

Reply:
This was from a merge after pulling from master

Reply:
https://github.com/department-of-veterans-affairs/vets-website/blob/5728ca8ade8a401dfc42ea69188c8770b3bdfc28/package.json#L255